providers/README.md

```diff
@@ -0,0 +1,10 @@
+# cloud-provider-gcp/providers
+
+Currently feature development is no longer accepted in `k8s.io/kubernetes/staging/src/k8s.io/legacy-cloud-providers/<provider>`. 
+This directory is to support further development for in-tree cloud provider gce.
```

Comment:
Pronouns are not your friend. Not clear without context, if "this" refers to the K/K staging directory or the directory your in. Better to be explicit. 

The *cloud-provider-gcp/providers* directory is to support further ...

Reply:
Thank you for pointing out. Updated